### PR TITLE
Case sensitive field names in sql queries #47

### PR DESCRIPTION
**Pull request by neokore**

### CODE REVIEW

**Comment by jesusbotella** [comment]
Just a silly question, but why are we adding double quotes if we changed all column names to lower case?

**Comment by VictorVelarde** [comment]
Not silly. The problem comes from the use of field names in **user datasets** like "My colum", "Lat point" and alike

File: packages/custom-storage/__tests__/SQLStorage.spec.ts [hunk 1]
```diff
@@ -186,9 +186,9 @@ describe('SQLStorage', () => {
   //     name: 'My Vis',
```
Comment (VictorVelarde) [inline packages/custom-storage/__tests__/SQLStorage.spec.ts]: let's remove the commented lines, if they are not useful

**Comment by VictorVelarde** [comment]
Ping me when it's ready, so I can rebase at metrics PR

**Pull request merged**